### DISCUSSION TITLE
Android原生低配的平板上卡顿

### DISCUSSION THREAD

**Discussion by Mr-Zhaozhi** [Tencent/libpag#2617] [Q&A] [answered]
`     
    implementation 'com.tencent.tav:libpag:4.3.51'//版本
    
    //使用方式 用的PagView 
 pagView.composition = PAGFile.Load(assets, "pag/1109003.pag")
            pagView.setRepeatCount(1)
            pagView.play()`
            很简单的使用；
            完整的序列帧动画 不是全屏 做出来的文件没有那么卡顿。
            用参数 输出了bmp的格式 全屏下的文件 在低配设备上卡顿。球高人指点

**Comment by kevingpqi123** [comment, marked as answer]
BMP 格式的文件本质上就是视频，低配的平板会有视频解码的压力，如果是中低端机型，优先使用纯矢量的素材，纯矢量的素材和 BMP 的素材性能相差是非常大的
